apps/src/applab/applab.js

```diff
@@ -6,7 +6,7 @@
  */
 
 'use strict';
-
+require('./mode-javascript_codeorg');
```

Comment:
I would suggest that this is moved somewhere else so that all apps that use droplet would have the proper file. studio, maze, and turtle wouldn't pick this up, but the change above to studioApp.js would mean that is expected to be loaded.